python/mxnet/ndarray.py

```diff
@@ -426,14 +440,27 @@ def _sync_copyfrom(self, source_array):
             ctypes.c_size_t(source_array.size)))
 
     def _slice(self, start, stop):
-        """Returns a sliced NDArray that shares memory with current one.
+        """Returns a sliced NDArray that shares memory with the current one.
+        This is called through x[start:stop].
 
         Parameters
         ----------
         start : int
-            Starting index of slice.
+            Starting inclusive index of slice in the first dim.
         stop : int
-            Finishing index of slice.
+            Finishing exclusive index of slice in the first dim.
+
+        Returns
+        -------
+            `NDArray` sharing the memory with the current one sliced from
+            start to end in the first dim.
```

Comment:
you mean "start" to "stop"? start and stop are both parameter names so they should be in single backticks.